### PR TITLE
Use single quotes

### PR DESCRIPTION
**Pull request by asi1024**
Use single quotes instead of double quotes.
Rel. cupy/cupy#700.

### CODE REVIEW

File: chainerx_build_helper.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def build_extension(self, ext):
             build_args += ext.build_targets
 
         env = os.environ.copy()
-        env['CXXFLAGS'] = '{} -DVERSION_INFO=\\"{}\\"'.format(
+        env['CXXFLAGS'] = '{} -DVERSION_INFO=\"{}\"'.format(
```
Comment (niboshi) [inline chainerx_build_helper.py]: Is this fix correct?

Comment (asi1024) [reply chainerx_build_helper.py]: Fixed.

**Comment by niboshi** [comment]
Jenkins, test this please

**Comment by asi1024** [comment]
Jenkins, test this please.

**Comment by niboshi** [comment]
LGTM

**Pull request merged**